doc/RecordingControl.xml

```diff
@@ -435,6 +440,52 @@ Change Request 2061, 2063, 2065, 2109</revremark>
           <emphasis role="bold">Destination</emphasis>: The destination is the track token of the track to which the device shall store the received data. All tracks must belong to the recording identified by the RecordingToken.</para>
         <para>The TrackInformation field for a Track holds a single Source. In case multiple RecordingJobs with differing Source are recording to the same Track it is undefined which of them is reported in the corresponding TrackInformation of the the RecordingSearch API.</para>
       </section>
+      <section>
+        <title>Event recording</title>
+        <para>A device signalling support for EventRecording via its capabilities shall support
+          controling recording job activity via the EventFilter with the following set of
+          parameters: 
+          </para>
+        <variablelist>
+          <varlistentry>
+            <term>TopicFilter</term>
+            <listitem>
+              <para>Topic filter as defined in section 9.6.3 of the ONVIF Core Specification.</para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>ContentFilter</term>
+            <listitem>
+              <para>Optional message content filter as defined in section 9.4.4 of the ONVIF Core
+                Specification. When this parameter is absent any message Data parameter with value
+                other than 'false', zero or empty shall result in a match. </para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>Before</term>
+            <listitem>
+              <para>Optional timespan to record before the actual event condition became active.
+              </para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>After</term>
+            <listitem>
+              <para>Optional timespan to record after the actual event condition becomes
+                inactive.</para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
+        <para>A device shall support Before and After durations when their limit is signalled via
+          the respective capability. A device may adapt Before and After duration values to internal
```

Comment:
Suggestion

"A device may adapt the Before and After duration values to internal quantization. </para>"